db/migrate/20241101154136_rename_img_url_to_gif_url_in_pokemons.rb

```diff
@@ -0,0 +1,5 @@
+class RenameImgUrlToGifUrlInPokemons < ActiveRecord::Migration[7.1]
```

Comment:
good design choice, helps us in the short term to know what we are working with and increases user empathy for future devs.